mentat/parsers/file_edit.py

```diff
@@ -83,7 +87,7 @@ async def is_valid(self) -> bool:
                     f"File {rel_path} does not exist, canceling all edits to file."
                 )
                 return False
-            elif rel_path not in code_file_manager.file_lines:
+            elif self.file_path not in code_context.include_files:
```

Comment:
When auto-context gets kicking we'll want a custom method for this (probably in code_context) that says:
- Check if there's at least one feature in code_context.features matching the path
- Check that one of those features contains all the lines flagged for editing here.

Just a heads up for now.